src/StatusQ/Components/StatusExpandableSettingsItem.qml

```diff
@@ -0,0 +1,150 @@
+import QtQuick 2.14
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+Rectangle {
+    id: root
+
+    property alias icon: icon
+    property alias primaryText: primaryText
+    property alias secondaryText: secondaryText
+    property alias button: button
+    property alias expandableComponent: expandableRegion.sourceComponent
+
+    property bool isExpandable: true
+    property bool hasButton: false
+    property  bool isExpanded: false
+    property bool hasDropDownArrow: false
+
+    anchors.left: parent.left
+    anchors.leftMargin: 20
+    anchors.right: parent.right
+    anchors.rightMargin: 20
+
+    radius: 8
+    color: Theme.palette.indirectColor1
```

Comment:
I don't think this is the correct color. In darkmode this has a way too high contrast.
<img width="793" alt="Screenshot 2021-09-02 at 10 49 17" src="https://user-images.githubusercontent.com/445106/131813440-a6bcd07c-672d-45ef-8193-be5b08fff464.png">

Reply:
in the design - https://www.figma.com/file/0m5JUlP8dv9szwQiGvP9gb/Wallet-Redesign-Sketchpad-%5BWIP%5D?node-id=357%3A33670 the value mentioned is -
background: #FFFFFF;

How do I get the right value for this? @PascalPrecht

Reply:
![image](https://user-images.githubusercontent.com/60327365/131909074-d6eb29b1-5bb5-49fc-882e-11b33fbddf34.png)
 This definitely looks better! @PascalPrecht

Reply:
Just to follow up here:

Sometimes, the the corresponding color of the dark theme doesn't really match what's needed in a design. E.g. `Theme.palette.indirectColor1` is white in light mode and black in darkmode. This works fine when the color is used for things like fonts.

When either mode's color doesn't match, we usually introduce a specific component-based mapping in `Theme.palette`. You can see a bunch of them for example here https://github.com/status-im/StatusQ/blob/master/src/StatusQ/Core/Theme/StatusDarkTheme.qml#L139-L180

Reply:
aah thanks.